packages/opentelemetry-metrics/src/BaseObserverMetric.ts

```diff
@@ -55,8 +55,12 @@ export abstract class BaseObserverMetric extends Metric<BoundObserver>
     );
   }
 
+  protected getObserverResult(): ObserverResult {
```

Comment:
I think the name should be something like `createObserverResult` as "get" suggests it will return something without really creating anything new or at least not every time.